sqlobject/src/main/java/org/jdbi/v3/sqlobject/statement/internal/ResultReturner.java

```diff
@@ -223,15 +223,15 @@ protected QualifiedType elementType(StatementContext ctx) {
         }
 
         @Override
-        protected QualifiedType elementType(StatementContext ctx) {
+        protected QualifiedType<?> elementType(StatementContext ctx) {
             return elementType;
         }
     }
 
-    static class SingleValueReturner extends ResultReturner {
-        private final QualifiedType returnType;
+    static class SingleValueReturner<T> extends ResultReturner {
```

Comment:
Any reason for introducing the `<T>` parameter on this and the next returner, but not the others?

Reply:
the others do `mapType` so become wildcards. Unless I'm mistaken

Reply:
`ConsumerResultReturner` doesn't get its type from a param, no room for a T
`VoidReturner` idem
`IteratorReturner` does `mapType`
`ResultIteratorReturner` idem
`StreamReturner` idem
`ResultIterableReturner` idem